docker-for-windows/wsl.md

```diff
@@ -77,6 +77,8 @@ Ensure you have completed the steps described in the Prerequisites section **bef
 
     Optionally, select any additional distributions you would like to enable WSL 2 on.
 
+    Please note that the WSL Integrations bits running in your distro depend on glibc. This make it difficult to run it on musl-based distros such as Alpine Linux.
```

Comment:
Are there any instructions to make it work? I think we should either say that it won't work, or here are the instructions, not just that it will be "difficult".

Reply:
Right. I am adding a link to the project we use in CI